portal/src/main/webapp/js/src/dynamicQuery.js

```diff
@@ -790,6 +841,9 @@ function geneSetSelected() {
 
         //  Set the gene list text area
         $("#gene_list").val(gene_set.gene_list);
+
+        // Validate the genes in the gene list
+        GeneSymbolValidator.initialize();
```

Comment:
This validator is only initialized when the user selects genes from the list "User-defined List". A gene that belongs to these pre-defined lists might not be in the database, and this case is not currently validated. That's why we're initializing the validator here, to check for these specific cases.